test/spot_feel/focus_test.rb

```diff
@@ -5,6 +5,8 @@
 module SpotFeel
   module Dmn
     describe :focus do
+      it "should should pass" do
+      end
```

Comment:
What's this guy for?

Reply:
Ah, I use that test when I'm working on a one-off problem. It's the same thing as using the 'focus' gem to only run a single test case.